presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -166,8 +166,21 @@ public void testRenameTable()
             throws Exception
     {
         assertQueryTrue("CREATE TABLE test_rename AS SELECT 123 x");
+
         assertQueryTrue("ALTER TABLE test_rename RENAME TO test_rename_new");
-        assertQueryTrue("DROP TABLE test_rename_new");
+        MaterializedResult materializedRows = computeActual("SELECT x FROM test_rename_new");
+        assertEquals(getOnlyElement(materializedRows.getMaterializedRows()).getField(0), 123L);
+
+        // provide new table name in uppercase
+        assertQueryTrue("ALTER TABLE test_rename_new RENAME TO TEST_RENAME");
+        materializedRows = computeActual("SELECT x FROM test_rename");
+        assertEquals(getOnlyElement(materializedRows.getMaterializedRows()).getField(0), 123L);
+
+        // test selecting from table with uppercase tableName
+        materializedRows = computeActual("SELECT x FROM TEST_RENAME");
```

Comment:
This seems like a general select test and belongs elsewhere (verify we already have a test that uses uppercase names)